jkube-kit/enricher/api/src/test/java/org/eclipse/jkube/kit/enricher/api/JKubeEnricherContextTest.java

```diff
@@ -220,7 +229,9 @@ void hasPlugin_withNullGroup_shouldSearchPluginWithArtifactId() {
   void hasPlugin_withGroup_shouldSearchPluginWithArtifactId() {
     try (MockedStatic<JKubeProjectUtil> jKubeProjectUtilMockedStatic = mockStatic(JKubeProjectUtil.class)) {
       // Given
-      JavaProject javaProject = mock(JavaProject.class, RETURNS_DEEP_STUBS);
+      JavaProject javaProject = JavaProject.builder()
+              .groupId("org.eclipse.jkube").artifactId("test-project").version("0.0.1")
+              .build();
```

Comment:
I see this is in multiple tests, Can this be moved in `@BeforeEach` clock?

Reply:
During squash the commits, I created multiple commits on this pull request, that's why I have created an updated pull request with these changes(#2619). Apologies for that.